packages/contracts-bedrock/scripts/generate-l2-genesis.sh

```diff
@@ -30,3 +32,16 @@ if [ ! -f "$OUTFILE_L2" ]; then
     --outfile.l2 "$OUTFILE_L2" \
     --outfile.rollup "$OUTFILE_ROLLUP" > /dev/null 2>&1
 fi
+
+# Wait for the L2 outfile to be over 8M for up to 2 seconds
+# This is a hack to ensure that the outfile is fully written
+# before the solidity tests try to read it
+for i in {1..8}; do
+  if [ $(du -m "$OUTFILE_L2" | cut -f1) -ge 8 ]; then
+    exit 0
+  fi
+  sleep 0.25
+done
+
+echo "L2 genesis file not generated in time. Exiting."
```

Comment:
Have you seen this string in the solidity code? I would need to double check that if writing non abi encoded data to standard out would break things. I specifically write all stdout/stderr to `/dev/null` to prevent this issue. I see CI passing and I am guessing it ends up hitting the `exit` case

Reply:
If the data isn't ABI encoded the script will fail, and print the output to the terminal. You can test this for yourself by moving the echo/exit 1 up in the script.